lib/libcc2/Map.cpp

```diff
@@ -340,6 +347,10 @@ bool cc2::Tile::haveLower(int type)
     case Bribe:
     case SpeedShoes:
     case Hook:
+    case UNUSED_94:
+    case UNUSED_9f:
+    case UNUSED_a3:
+    case UNUSED_f3:

```

Comment:
The trouble with "supporting" these (and why I didn't do it originally) is that the tile definitions in CC2 are essentially reading "random" data beyond the end of the array, meaning that these behaviors change from one version to another of CC2.  You can already see this by checking out older releases of CC2 and observing that tiles beyond Hook load differently than they do in the latest release.  If we *really* want to support extended tiles, we'll need to have some sort of executable version support added as well :/